src/plugins/plot/pluginSpec.js

```diff
@@ -410,6 +410,20 @@ describe("the plugin", function () {
             expect(options[1].value).toBe("Another attribute");
         });
 
+        it("Updates the Y-axis label when changed", () => {
```

Comment:
@shefalijoshi Let me know if this test makes sense!

Reply:
This looks good. Thanks!